nixos/modules/services/networking/stunnel.nix

```diff
@@ -16,8 +16,12 @@ let
   serverConfig = {
     options = {
       accept = mkOption {
-        type = types.int;
-        description = "On which port stunnel should listen for incoming TLS connections.";
+        type = types.either types.str types.int;
+        description = ''
+	  On which [host:]port stunnel should listen for incoming TLS connections.
+	  Note that unlike other softwares stunnel ipv6 address need no brackets,
+	  so to listen on all IPv6 addresses on port 1234 one would use ':::1234'.
+	'';
```

Comment:
Thanks for the documentation, this looks great!

Would you mind fixing the indentation here? Essentially the multi-line string start token should be on the same level as the end token. The white space in front should then be removed automatically.

Reply:
Are you sure? I see the indentation I used all over the place (`git grep -A 5 "description = ''"`)
I had disabled documentation building but will turn it back on to check, will repost in a few minutes

Reply:
I'm quite certain Nix will strip the whitespace. It might be nitpicking, though I think this identation style is more common (and more pleasing to look at):

```
$ cat test.nix 
let 
  a = ''
    test
  '';

in
  a

$ nix repl
Welcome to Nix version 2.3.6. Type :? for help.

nix-repl> import ./test.nix
"test\n"
```

Reply:
Oh, I just now realize you've been using tabs, which render for me as 2 spaces, though probably will render as 8 spaces for you. :) Given that the rest of the file uses spaces, that might be more reliable.